packages/botonic-core/src/types/constants.d.ts

```diff
@@ -0,0 +1,14 @@
+export const Providers: Readonly<{
+  Messaging: {
```

Comment:
Should this field name be lowercase?

Reply:
yep, we should consider how to name this in Botonic 1.0, atm, some packages are using it, so we should maintain it like this just to not break things.